docs/index.rst

```diff
@@ -24,23 +24,17 @@ Table of Contents
 =================
 
 .. toctree::
-    :maxdepth: 4
-    :numbered:
-
-    ptoled
-    keyboard
-    ptbuttons
-    pulse/eeprom
-    pulse/manual-configuration
-
-    input_components
-    output_components
-
-    component_examples
-    basic_examples
-    advanced_examples
-
-    parameters
+    :maxdepth: 2
+
+    installing
+    sdk_reference
+    cli
+    examples
```

Comment:
After "getting started", I think we should lead with examples Fastest way to communicate what it can do!

Reply:
How will this be different from the examples provided in each section?